ingestion/setup.py

```diff
@@ -179,7 +179,7 @@ def get_long_description():
     "kinesis": {VERSIONS["boto3"]},
     "ldap-users": {"ldap3==2.9.1"},
     "looker": {"looker-sdk>=22.20.0"},
-    "mlflow": {"mlflow-skinny~=1.30"},
+    "mlflow": {"mlflow-skinny~=1.30", "alembic==1.10.2"},
```

Comment:
`alembic` was giving not found error for mlflow. Added it here in case it was missed

Reply:
can we keep this a bit laxer? `~=`

Reply:
Sure I'll do that